src/global/variables/clauses/IntegerLiteral.java

```diff
@@ -1,26 +1,81 @@
 package global.variables.clauses;
 
+import global.tools.InvalidClauseException;
+import global.utils.RegexUtil;
 import global.variables.Clause;
 
 public class IntegerLiteral extends Clause {
-    int value;
+    int value = 0;
```

Comment:
Can't we just leave this null? I think everything should be null unless set explicitly in a constructor